src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature.disabled

```diff
@@ -0,0 +1,50 @@
+Feature: Discovery API that retrieves the information from the base endpoint of the API
+
+        @ignore
+        Scenario: 01 Retrieves information of the API
+             Given a get to the root of the API
+             Then it should respond with 200
+              And the response body is
+                  """
+                  {
+                      "openapi": "3.1.0"
+                      "info": {
+                          "version": "1.0.0",
+                          "applicationName": "Ed-Fi Alliance Data Management Service"
+                      }
+                      "dataModels": [
+                          {
+                              "name": "Ed-Fi",
+                              "version": "5.1.0",
+                              "informationalVersion": "The Ed-Fi Data Standard v5.1"
+                          }
+                      ],
+                      "urls": [
+                          {
+                              "dependencies": "https://api.ed-fi.org/DMS/metadata/dependencies"
```

Comment:
This is tricky, because the base URL will not be `https://api.ed-fi.org/DMS`. The actual value will depend on how how the system is called. Let's put in a place holder, replacing `https://api.ed-fi.org/DMS` with `{BASE_URL}` everywhere.

Reply:
Changed